src/clj/velho_ds/macros.clj

```diff
@@ -0,0 +1,8 @@
+(ns velho-ds.macros)
+
+(defmacro $-> [input]
```

Comment:
The macro could be named to something a bit less cryptic. E.g. component-and-listing. Such something that makes the calls explicitly state their intent.

Reply:
That is fine by me. The idea behind that supershort name is that it should not be too verbose to call this "show example function". That's because I think the styleguide part of the system should not be intrusive for developers to use. It should be "as easy as the real thing" to create an example out of the component or there lies a risk that developers don't want to do these components to the library.

Reply:
Agreed. That's a good point. Let's go with the short name for now.